I added a comment explaining the rationale for `disallow_untyped_defs = false`: https://github.com/vippsas/zeroeventhub/blob/678b7740320130a7f2a5dd27e2e9e065818cc8b7/python/zeroeventhub/pyproject.toml#L51-L55

To illustrate the problem:

```console
$ git diff
diff --git a/python/zeroeventhub/pyproject.toml b/python/zeroeventhub/pyproject.toml
index 82996a9..7f91d72 100644
--- a/python/zeroeventhub/pyproject.toml
+++ b/python/zeroeventhub/pyproject.toml
@@ -48,7 +48,7 @@ ignore_missing_imports = true

 [[tool.mypy.overrides]]
 module = "tests.*"
-disallow_untyped_defs = false
+# disallow_untyped_defs = false
 check_untyped_defs = true

 [tool.pylint.parameter_documentation]
$ poetry run mypy
tests/test_page_event_receiver.py:12: error: Function is missing a return type annotation  [no-untyped-def]
tests/test_page_event_receiver.py:27: error: Function is missing a type annotation  [no-untyped-def]
tests/test_page_event_receiver.py:58: error: Function is missing a type annotation  [no-untyped-def]
tests/test_page_event_receiver.py:76: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:20: error: Function is missing a return type annotation  [no-untyped-def]
tests/test_client.py:25: error: Function is missing a return type annotation  [no-untyped-def]
tests/test_client.py:38: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:65: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:85: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:109: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:138: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:167: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:194: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:221: error: Function is missing a type annotation  [no-untyped-def]
tests/test_client.py:257: error: Function is missing a type annotation  [no-untyped-def]
Found 15 errors in 2 files (checked 9 source files)
```

--- a/python/zeroeventhub/pyproject.toml
+++ b/python/zeroeventhub/pyproject.toml
@@ -48,7 +48,7 @@ ignore_missing_imports = true
 
 [[tool.mypy.overrides]]
 module = "tests.*"
-# `disallow_untyped_defs` has to be explicitly disabled because it is enbaled by
+# `disallow_untyped_defs` has to be explicitly disabled because it is enabled by
 # `strict`, and just setting `check_untyped_defs` does not disable it, even
 # though it is from the same option group as `check_untyped_defs`. With it
 # enabled Mypy complains about test functions without type annotations.

